### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by techfreek**
I know it's only two commits, but here is what this does:
* Adds the ipfs logo as the favicon to ipfs.io and ipn.io
* Creates multiple sizes of the alpha-sunrise image used on ipfs.io so small devices don't have to load a 2000x1682 image
* Added a background color behind the alpha-sunrise picture so the text is readable as the picture loads
* Adds a missing npm module to the package.json file
* Adds a title to ipfs.io and ipn.io
* Site specific css files (Mostly just for differences in the header picture)

Does not:
*  try to improve publishing

Feel free to make comments.

### CODE REVIEW

File: ipn.io/src/ipn.io/index.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 baseurl: ./
-mainimg: ipn-sunrise.png
+title: Interplanetary Networks, Inc.
 tagline: Interplanetary Networks, Inc.
 template: tmpl/layouts/ipn.html
+site: ipn
```
Comment (jbenet) [inline ipn.io/src/ipn.io/index.html]: does the `site` identifier need to go on all the sources now? otherwise wont be picked up everywhere

**Comment by jbenet** [comment]
@techfreek thanks for the PR -- this is great. Everything LGTM-- just note about `site` above-- not sure it will work.

**Comment by jbenet** [comment]
(i'll go through and republish as it's a bit cumbersome atm-- i've to go pin data manually)

**Comment by techfreek** [comment]
That's a fair point about the site tag. I didn't carry those changes down to the subpages. I can fix that later tonight if you want to hold off on republishing

**Comment by jbenet** [comment]
thanks@

**Pull request merged**